runtime/altair/src/migrations.rs

```diff
@@ -10,14 +10,47 @@
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 // GNU General Public License for more details.
 
-use crate::{ForeignInvestments, OraclePriceCollection, OraclePriceFeed, OrderBook};
+use cfg_primitives::AccountId;
+use sp_core::parameter_types;
+
+parameter_types! {
+	pub InitialTcMembers: sp_std::vec::Vec<AccountId> = sp_std::vec![
```

Comment:
NOTE: Missing 4 members at the time of writing. I'd like this PR to get merged rather sooner than later such that a subsequent PR to add the missing ones is preferred from my side. But no strong opinion.